src/DynamicData/List/ObservableListEx.cs

```diff
@@ -957,6 +957,186 @@ public static IObservable<IEnumerable<T>> LimitSizeTo<T>(this ISourceList<T> sou
         return new MergeMany<T, TDestination>(source, observableSelector).Run();
     }
 
+    /// <summary>
+    /// Operator similiar to Merge except it is ChangeSet aware.  All of the observable changesets are merged together into a single stream of ChangeSet events.
+    /// </summary>
+    /// <typeparam name="TObject">The type of the object.</typeparam>
+    /// <param name="source">The Source Observable ChangeSet.</param>
+    /// <returns>The result from merging the changesets together.</returns>
+    /// <exception cref="ArgumentNullException">Parameter was null.</exception>
+    public static IObservable<IChangeSet<TObject>> MergeChangeSets<TObject>(this IObservable<IObservable<IChangeSet<TObject>>> source)
+        where TObject : notnull
+    {
+        source.ThrowArgumentNullExceptionIfNull(nameof(source));
+
+        return new MergeChangeSets<TObject>(source, equalityComparer: null).Run();
+    }
+
+    /// <summary>
+    /// Operator similiar to Merge except it is ChangeSet aware.  All of the observable changesets are merged together into a single stream of ChangeSet events.
+    /// </summary>
+    /// <typeparam name="TObject">The type of the object.</typeparam>
+    /// <param name="source">The Source Observable ChangeSet.</param>
+    /// <param name="equalityComparer"><see cref="IEqualityComparer{T}"/> instance to determine if two elements are the same.</param>
+    /// <returns>The result from merging the changesets together.</returns>
+    /// <exception cref="ArgumentNullException">Parameter was null.</exception>
+    public static IObservable<IChangeSet<TObject>> MergeChangeSets<TObject>(this IObservable<IObservable<IChangeSet<TObject>>> source, IEqualityComparer<TObject> equalityComparer)
```

Comment:
We could but the more optional parameters you have, the more complicated it is to use if you want to supply the one on the end.  I'm not sure the best way to handle this.  It easier to have more optional parameters, but I think more functions is a better user experience.

If we decide to remove the other parameters, then this definitely could be made optional.  That would simplify a lot, actually, but it would make List and Cache asymmetrical.

Reply:
> the more complicated it is to use if you want to supply the one on the end

Maybe I'm misunderstanding, but how is this complicated at all? Invoke with parameter names, and you're good, which is really the strategy you should be using _any_ time optional parameters are involved.

> I think more functions is a better user experience

I disagree entirely. Optional parameters are way better for my tastes. As a consumer, they make methods easier to approach, on account of there being fewer overloads to look at to get the full picture of what a method can do, they make method with many overloads easier to understand overall, because there's less redundant documentation to read through, and they make methods easier to invoke, by eliminating weird binding issues, you just supply only the parameters you want, by name, and you're done.

> it would make List and Cache asymmetrical

Intuitively, I don't see this as a big issue, List and Cache are already too fundamentally different, by nature.

I had a disucssion with @glennawatson in Slack, and I think the main takeaway is that you should use avoid optional parameters only when it's a breaking-change issue, I.E. you're considering adding parameters to existing methods. That seems to be Microsoft's general approach.

Reply:
I can make it an optional parameter and we'll see how that goes.

Reply:
Now there are fewer overloads... Hopefully that makes the one someone needs easier to find.